src/capi_frontend/capi_utils.cpp

```diff
@@ -145,7 +145,7 @@ Status prepareConsolidatedTensorImpl(InferenceResponse* response, const std::str
     Status status = response->addOutput(
         name,
         getPrecisionAsOVMSDataType(ovElementTypeToOvmsPrecision(precision)),
-        shape.data(),
+        reinterpret_cast<const int64_t*>(shape.data()),  // signed unsigned issue?
```

Comment:
Is this TODO to resolve in this MR?

Reply:
The issue remains the same for all frontends - tfs/kfs/c-api during serialization.